src/tokenizers/config.rs

```diff
@@ -0,0 +1,47 @@
+use crate::error::BoxedError;
+use std::path::PathBuf;
+
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(super)))]
+pub enum TokenizerConfigError {
+    #[snafu(display("Couldn't resolve filepaths in tokenizer configuration"))]
+    ResolvePaths { source: BoxedError },
+
+    #[snafu(display("Cannot open tokenizer config for reading: {path:?}"))]
```

Comment:
`to_string_lossy` gives nicer output (not quoted): https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=30d46b5123a1dc7d16d384301950cdbf

Reply:
Fixed.